recipes/synthpop/LICENSE.txt

```diff
@@ -0,0 +1,26 @@
+Copyright (c) 2016, UrbanSim Inc. All rights reserved.
```

Comment:
Can you open an issue upstream about the license file missing in sdist?

Reply:
Good idea. I just added a MANIFEST file, which should take care of it when the next release comes out: https://github.com/UDST/synthpop/commit/f4800870291490f38af2e4b2be73222c5c24d353